### PR TITLE
chore: Mention contributors in app info

### PR DESCRIPTION
**Pull request by mvorisek**
Like in phpunit header: `PHPUnit 10.5.5 by Sebastian Bergmann and contributors.`.

~Also mention contributors in all file headers.~

### CODE REVIEW

**Comment by Wirone** [comment]
@mvorisek would be better if you reconfigured header and applied fix in a separate commits, because it's hard to grasp the core changes because of chamged header in all the files..

**Comment by mvorisek** [comment]
> @mvorisek would be better if you reconfigured header and applied fix in a separate commits, because it's hard to grasp the core changes because of chamged header in all the files..

repushed

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
     (c) Fabien Potencier <fabien@symfony.com>
         Dariusz Rumiński <dariusz.ruminski@gmail.com>
+        and contributors <https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/graphs/contributors>
```
Comment (keradus) [inline .php-cs-fixer.dist.php:21]: I would not like to change the license header.

for what it's worth, I also do not see big projects doing so [eg](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Asset/Package.php), [eg](https://github.com/sebastianbergmann/phpunit/blob/main/src/Exception.php)

**Review by keradus — CHANGES_REQUESTED** [review]
.

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (mvorisek) [reply .php-cs-fixer.dist.php:21]: I would say it is never enough to give credit back to the community. This tools is definitely not authored/copyrighted by 2 people only.

Comment (keradus) [reply .php-cs-fixer.dist.php:21]: Let's widen it to most open source projects and we are good to go ;)

Contributions are definitely welcome and appreciated, but if you pick up the fight to have all license headers mentioning contributors, please start with biggest projects, let's take it as my delegation to have it concluded first on, let say, example Symfony or PHPUnit. 

For now, I'm more than happy to merge this PR limited to first commit.

Comment (mvorisek) [reply .php-cs-fixer.dist.php]: done

**Review by keradus — APPROVED** [review]

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (Wirone) [reply .php-cs-fixer.dist.php]: I never understand the need to compare with others when it comes to non-technical things. Does it matter if Symfony includes "and all contributors" in a header, when it comes to this repo? I would understand the reasoning related to law, for example "if we add it to the header then spread decisiveness across all contributors" (I don't know if it's the case, it's just an example), but I don't get why bigger projects should do something first 🤷‍♂️🤯. Can't Fixer be a precursor for such things?

I agree with @mvorisek that _a lot_ of other people added _a lot_ to this codebase and referencing them explicitly as a group would be a really nice move.

But it's only my 2 cents.

Comment (keradus) [reply .php-cs-fixer.dist.php]: license header or license itself is always tricky. i recall the case was tricky already for few bigger repos, including repos with big companies behind it, some even went crazy with extra sign-off of move the "all rights" to repo owner. I do not have big legal team, i do not have big time to investigate myself, and if it's idea I'm not convinced myself and I do not find it as industry standard, I'm not comfortable to make this move or to invest capacity into it.

same time, contributors are appreciated and it was no brainer to add them to all tool usage.

Comment (Wirone) [reply .php-cs-fixer.dist.php]: And this explanation provides better context, which I understand. Thanks for clarifying.

**Pull request merged**